packages/esm-ohri-pmtct-app/src/pmtct/patient-chart/mch-summary/tabs/current-pregnancy.component.tsx

```diff
@@ -274,17 +272,7 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid, pTrackerId
         />
       </div>
 
-      {motherPreviousVisitTabs.map((tab) => (
-        <EncounterList
-          patientUuid={patientUuid}
-          encounterType={tab.encounterType}
-          columns={tab.columns}
-          description={tab.description}
-          headerTitle={tab.headerTitle}
-          formList={tab.formList}
-          launchOptions={tab.launchOptions}
-        />
-      ))}
+      <EncounterListTabsComponent patientUuid={patientUuid} configSchema={motherPreviousVisitConfigSchema} config={config} />
```

Comment:
I also realised you didn't do the same for `hivExposedInfant`, so let's have a sync about this and see how we wanna handle that

Reply:
> @CynthiaKamau this might not work because this wasn't a tab, we had such cases in the covid app and for now I would say we just leave it as it is unless we decide to render everything as a tab. if you see this has no tabs or tabPanels but if we treat it like this, it will have that blue tab design, we need to make a decision about this, personally I would say we make every menu that renders the encounter list table a tab. I don't know what you think. cc @lucyjemutai

It is a list of tabs
<img width="1469" alt="Screenshot 2024-07-08 at 10 06 34" src="https://github.com/UCSF-IGHS/openmrs-esm-ohri/assets/19533785/72231ab6-186f-494c-b4bb-32df804f18b5">

Reply:
How is it working because in the code it doesn't look like that, we were just rendering the Encounter list